app/assets/stylesheets/action_page.scss

```diff
@@ -1,30 +1,9 @@
 /*
- * This is a manifest file that'll be compiled into action_page.css, which will include all the files
- * listed below.
- */
-
-.redtop {
-  padding: 2em 0 0;
-  h1 {
-    font-size: 2.5em;
-    font-weight: normal;
-    line-height: 1.2;
-    margin-top: 0;
-  }
+* This is a manifest file that'll be compiled into action_page.css, which will include all the files
```

Comment:
It's probably better to remove these comments rather than update them. I'm not sure where they came from originally, but it's not true anymore -- the only files compiled into .css are application.scss and the stylesheets listed in `Rails.application.config.assets.precompile`.

Reply:
agreed.